core/scheduler/scheduler.go

```diff
@@ -213,11 +239,13 @@ func (s *Scheduler) scheduleSlot(ctx context.Context, slot core.Slot) {
 		}()
 	}
 
+	// Resolve sync committee duties for the next epoch if the current slot is second last in the current epoch.
+	if slot.LastNInEpoch(2) {
```

Comment:
Rather not introduce this complexity, keep to existing logic and patterns. We can improve the logic in a separate PR to do lookaheads, for v1, just do current slot/epoch.